topics/benchmark.md

```diff
@@ -6,13 +6,28 @@ description: >
 
 ## Usage
 
-**`valkey-benchmark`** [ _OPTIONS_ ] [ _COMMAND_ _ARGS_... ]
+**`valkey-benchmark`** [ _OPTIONS_ ] [**`--`**] [ _COMMAND_ _ARGS_... ]
 
 ## Description
 
-Valkey includes the `valkey-benchmark` utility that simulates running commands done
-by N clients while at the same time sending M total queries. The utility provides
-a default set of tests, or you can supply a custom set of tests.
+Simulates sending commands using multiple clients. The utility provides a
+default set of tests. You can run a subset of the tests using the `-t` option or
+supply one or more custom commands on the command line.
+
+To supply a multiple commands on the command line, separate them with `;`
+(quoted as `';'` to protect it from expansion by the shell).
```

Comment:
I know there is an example later on, but IMO placing an example here might help clarify the explanation